tests/fixtures/sourcemaps/08_malformed_abs_path/files.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
do we even need an empty file here? maybe in a different test we should add a test for failing to fetch the manifest

Reply:
We need it for now so that `sourcemap_router` wont break😅